config-linux.md

```diff
@@ -330,14 +330,19 @@ The following parameters can be specified to set up the controller:
 
 * **`weight`** *(uint16, OPTIONAL)* - specifies per-cgroup weight. This is default weight of the group on all devices until and unless overridden by per-device rules.
 * **`leafWeight`** *(uint16, OPTIONAL)* - equivalents of `weight` for the purpose of deciding how much weight tasks in the given cgroup has while competing with the cgroup's child cgroups.
-* **`weightDevice`** *(array of objects, OPTIONAL)* - specifies the list of devices which will be bandwidth rate limited. The following parameters can be specified per-device:
+* **`weightDevice`** *(array of objects, OPTIONAL)* - specifies the list of devices which will be weight division of bandwidth. The following parameters can be specified per-device:
     * **`major, minor`** *(int64, REQUIRED)* - major, minor numbers for device. More info in [mknod(1)][mknod.1] man page.
-    * **`weight`** *(uint16, OPTIONAL)* - bandwidth rate for the device.
-    * **`leafWeight`** *(uint16, OPTIONAL)* - bandwidth rate for the device while competing with the cgroup's child cgroups, CFQ scheduler only
+    * **`weight`** *(uint16, OPTIONAL)* - weight division of bandwidth for the device.
+    * **`leafWeight`** *(uint16, OPTIONAL)* - weight division of bandwidth for the device while competing with the cgroup's child cgroups, CFQ scheduler only
 
     You MUST specify at least one of `weight` or `leafWeight` in a given entry, and MAY specify both.
 
-* **`throttleReadBpsDevice`**, **`throttleWriteBpsDevice`**, **`throttleReadIOPSDevice`**, **`throttleWriteIOPSDevice`** *(array of objects, OPTIONAL)* - specify the list of devices which will be IO rate limited.
+* **`blkioThrottleReadBpsDevice`**, **`blkioThrottleWriteBpsDevice`** *(array of objects, OPTIONAL)* - specify the list of devices which will be bandwidth rate limited.
+    The following parameters can be specified per-device:
+    * **`major, minor`** *(int64, REQUIRED)* - major, minor numbers for device. More info in [mknod(1)][mknod.1] man page.
+    * **`rate`** *(uint64, REQUIRED)* - bandwidth rate limit in bytes per second for the device
+
+* **`blkioThrottleReadIOPSDevice`**, **`blkioThrottleWriteIOPSDevice`** *(array of objects, OPTIONAL)* - specify the list of devices which will be IO rate limited.
     The following parameters can be specified per-device:
     * **`major, minor`** *(int64, REQUIRED)* - major, minor numbers for device. More info in [mknod(1)][mknod.1] man page.
     * **`rate`** *(uint64, REQUIRED)* - IO rate limit for the device
```

Comment:
If we are not going to require these to be implemented by `blkio.throttle.read_iops_device` and `blkio.throttle.write_iops_device`, I suggest we at least inline the [kernel's docs for those settings][1]:

> Specifies upper limit on read (or write, for `blkioThrottleWriteIOPSDevice`) rate to the device. IO rate is specified in io per second. Rules are per device.

Although I'm not sure how much we can legally copy from the GPLv2 kernel docs into this Apache-2 project.  I'd much rather require these to have the same semantics as `blkio.throttle.read_iops_device` and `blkio.throttle.write_iops_device`, link to the kernel docs, and leave it at that.

[1]: https://www.kernel.org/doc/Documentation/cgroup-v1/blkio-controller.txt